tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1059,6 +1250,28 @@ static uint8_t stop_discovery(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
+#if defined(CONFIG_BT_CLASSIC)
+static uint8_t br_connect(const struct btp_gap_connect_cmd *cp)
+{
+	struct bt_conn *conn;
+
+	conn = bt_conn_create_br(&cp->address.a, BT_BR_CONN_PARAM_DEFAULT);
+	if (!conn) {
```

Comment:
Expressions in `if` statements shall be boolean as per rule 85 of https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html

```suggestion
	if (conn == NULL) {
```

Reply:
Updated.